.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  "plugins": [
+    "codeceptjs"
+  ],
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es2020": true,
+    "codeceptjs/codeceptjs": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 11
+  },
+  "rules": {
```

Comment:
how about adding
```
"semi": [
  "error",
  "always"
]
```

Reply:
sure, there are more rules I am sure we can add too as we go